Makefile

```diff
@@ -118,6 +118,6 @@ lint:
 .PHONY: lint-fix
 lint-fix:
 	python3 -c "import black" > /dev/null 2>&1 || python3 -m pip install black
-	python3 -c "import isort" > /dev/null 2>&1 || python3 -m pip install isort
 	python3 -m black --target-version py38 .
-	python3 -m isort .
+	python3 -c "import ruff" > /dev/null 2>&1 || python3 -m pip install ruff
+	python3 -m ruff --preview --fix .
```

Comment:
If stability is a concern, then wouldn't it be better to not use `--preview`? I have to imagine that any features that were in preview back when this PR was created are no longer in preview.

Reply:
Sans `--preview`:

```
warning: Selection `LOG` has no effect because the `--preview` flag was not included.
warning: Selection `LOG` has no effect because the `--preview` flag was not included.
warning: Selection `LOG` has no effect because the `--preview` flag was not included.
```

Reply:
(LOG was added via #7421.)

Reply:
I think we should go for stability and avoid `--preview`. Any idea when LOG will leave preview?

Reply:
I don't really think there's a big huge stability issue here, since whoever ends up updating the Ruff version in this repo will probably look at what they're doing?

Also, the more happy users a `preview` rule has, the better its chances are to graduate, I think 😁

Reply:
Let's remove `--preview`, I'd prefer to keep things stable and not spend time double checking potentially-unstable changes every month.

Reply:
Removed `--preview`, and also the `"LOG"` family of lints, because it would otherwise warn about it on every `--preview`less invocation.